src/vs/monaco.d.ts

```diff
@@ -34,12 +34,25 @@ declare namespace monaco {
 		 */
 		getWorkerUrl?(workerId: string, label: string): string;
 		/**
-		 * Create a trust types policy (same API as window.trustedTypes.createPolicy)
+		 * Create a trusted types policy (same API as window.trustedTypes.createPolicy)
 		 */
-		createTrustedTypesPolicy<Options extends TrustedTypePolicyOptions>(
+		createTrustedTypesPolicy(
```

Comment:
@alexdima Shouldn't this method be optional? `createTrustedTypesPolicy?()`

Because of this change, Monaco Editor 0.39 requires this method to be implemented when creating a new environment:

https://microsoft.github.io/monaco-editor/docs.html#interfaces/Environment.html

This "breaks" type-checking in some Monaco Editor examples:

https://github.com/microsoft/monaco-editor/blob/12a85be518baa63571b5ccf780cfeec12b0ff2b4/samples/browser-esm-webpack/index.js#L3-L19

```
Error: Property 'createTrustedTypesPolicy' is missing in type '{ getWorkerUrl: (_workerId: string, label: string) => string; }' but required in type 'Environment'. (ts)
```

Reply:
Created https://github.com/microsoft/vscode/pull/185454 with a fix.